### ISSUE TITLE
Points not added to ps_gamifications_customer table

### ISSUE THREAD

**Issue by maserman** [sarjon/gamifications#7]
Have installed module on Prestashp v 1.7.4.4 and when all order statuses are enabled under Loyalty -> shopping points completed transactions fail to add points in table for customers.

I've tried manually adding points in that table and then Vouchers can be created and used as rewards fine.

Just seems to be an issue with automatically creating points on orders.

Is there a fix?

Thanks
Baz

**Comment by Berna-JGAC** [comment]
Yes. There are no points added to the customer. Is there an available fix? Thank you.

**Comment by Setax2** [comment]
There are no points added here too. Could you help us please?

**Comment by chivomd** [comment]
The problem is in `class GamificationsShoppingPointActivity` on function `rewardCustomer` line 88. This call `$order->getTotalPaid($defaultCurrency)` returns an amount of zero.
If you replace `$totalPaid = $order->getTotalPaid($defaultCurrency);` by`$totalPaid = $order->total_products_wt;` it woks.
And adding `$earnedPoints` to `GamificationsActivityHistory::log` as the fourth parameter on line 125 makes reward history to show how many points are obtained with that order.
